posts/an-introduction-to-dqm.md

```diff
@@ -0,0 +1,63 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+date: 2020-06-15T14:50:55.000Z
+title: "An Introduction to Data Quality Management (DQM)"
+image:
+---
+
+# What is Data Quality Management (DQM)?
+Data Quality Management (DQM) is a practice that aims to maintain a high quality of data. DQM extends from data acquisition to distribution of data. DQM is about employing processes, methods, and technologies to ensure the quality of the data meets specific business requirements. DQM is extensive, with far-reaching results, if applied in a reliable, consistent, continual manner. Ultimately, DQM aims to deliver trusted data promptly.
+
+# Why do you need Data Quality Management Processes in Place?
+Most of a company's operations and decisions rely heavily on data, so the emphasis on data quality is even higher. Data quality refers to the assessment of information collected relative to its purpose and its ability to serve that purpose. Low-quality data costs organizations daily. An estimate by IBM calculated that the annual cost of data quality-related issues amounted to around [$3.1 trillion](https://www.ibmbigdatahub.com/infographic/four-vs-big-data).
+
+Good data quality increases the accuracy of analytics applications, which can lead to better business decision-making that boosts sales and improves internal processes. High-quality data can help expand the use of BI dashboards and analytics tools, as well. If analytics data is seen as trustworthy, business users are more likely to rely on it instead of basing decisions on their gut feelings or spreadsheets.
```

Comment:
```suggestion
Good data quality increases the accuracy of analytics applications, leading to better business decision-making that boosts sales and improves internal processes. High-quality data can help expand the use of BI dashboards and analytics tools, as well. If business users see analytics data as trustworthy, they are more likely to rely on it instead of basing decisions on their gut feelings or spreadsheets.
```